src/Resources/RoleResource/RelationManager/UserRelationManager.php

```diff
@@ -58,7 +58,7 @@ public function table(Table $table): Table
             ->columns([
                 TextColumn::make(config('filament-spatie-roles-permissions.user_name_column'))
                     ->label(__('filament-spatie-roles-permissions::filament-spatie.field.name'))
-                    ->searchable(config('filament-spatie-roles-permissions.user_name_searchable_columns')),
+                    ->searchable(config('filament-spatie-roles-permissions.user_name_searchable_columns') ?? true),
```

Comment:
While `?? true` works, would it be better to pass the default of `, true` to the `config()` call instead?
```suggestion
                    ->searchable(config('filament-spatie-roles-permissions.user_name_searchable_columns', true)),
```

Both ways should work. Just asking whether we want to use a Laravel convention, or null-coalesce. 
I have no strong opinion/preference.

Reply:
Might be this is a better way, but as for me, I thought that `true` is not the best fit as a default value for config. But will work as well, you are right